Services/DataBaseService/Mappers/UserMapper.cs

```diff
@@ -31,7 +31,13 @@ public DbUserCredential MapToDbUserCredential(UserCredential userCredential)
 
         public User MapUser(DbUser dbUser)
         {
-            throw new NotImplementedException();
+            return new User

```

Comment:
User doesn't have email here?

Reply:
Currently dbUser does not have an email address and we discussed the option to get email on the client.
If necessary, we can add it here, but it will be a little more difficult

Reply:
Why we should save it on client?

Reply:
Because this information can be necessary for another actions.
I'll check with management)

Reply:
This issue has already been resolved